src/Symfony/Component/HttpClient/Tests/Response/MockResponseTest.php

```diff
@@ -7,7 +7,7 @@
 use Symfony\Component\HttpClient\Response\MockResponse;
 
 /**
- * Test methods from Symfony\Component\HttpClient\Response\ResponseTrait.
+ * Test methods from Symfony\Component\HttpClient\Response\*ResponseTrait.
```

Comment:
Isn't the star too much?

Reply:
`*ResponseTrait` = CommonResponseTrait + TransportResponseTrait